backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunServiceImpl.java

```diff
@@ -2410,7 +2428,13 @@ public void commitMultipartArtifact(
       }
 
       if (errorMessage != null) {
-        throw new ModelDBException(errorMessage, io.grpc.Status.Code.INVALID_ARGUMENT);
+        throw new ModelDBException(errorMessage, Code.INVALID_ARGUMENT);
+      }
+
+      if (App.getInstance().getTrialEnabled()) {
+        throw new ModelDBException(
+            "Multipart artifact upload not supported on the trial version",
+            Code.FAILED_PRECONDITION);
       }
```

Comment:
Follow the format for errot messages and error code

Reply:
it is already resolved on other subsequent PR https://github.com/VertaAI/modeldb/pull/1616